cypress/e2e/patrimoine.cy.js

```diff
@@ -12,41 +12,54 @@ import "cypress-axe"
 context("Full simulation", () => {
   beforeEach(() => {
     navigate.init()
+    navigate.goToIframe()
     cy.injectAxe()
+    cy.get('iframe[id="simulateur"]', { timeout: 7000 }).as("iframe")
   })
 
-  it("accept a basic situation", () => {
-    navigate.goHome()
-
-    profil.defaultIndivu()
-
-    foyer.children(0)
-    foyer.fill_en_couple(false)
-
-    logement.fill__logementType("locataire")
-    logement.fill__locationType("vide")
-    logement.fill_coloc(false)
-    logement.fill_logement_chambre(false)
-    logement.fill_proprietaire_proche_famille(false)
-    logement.fill_loyer(600, 100)
-    logement.fill_depcom("34000")
-    logement.fill__nombreMoisEntreeLogement(-2)
-
-    revenu.fill_ressources_types(["salaire_net"])
-    revenu.fillConstantRevenu(400)
-
-    projet.fill__interetsAidesVelo(["velo_mecanique"])
-    projet.fill__interetBafa(false)
-    projet.fill__interetPermisDeConduire(false)
-    projet.fill__interetAidesSanitaireSocial(false)
-
-    results.wait()
-    results.hasAideVeloNationale()
-    results.hasHousingBenefit()
-    results.back()
-    results.hasRSA()
-    form.fillPatrimoine()
-    results.wait()
-    results.hasNotBenefit("rsa")
+  it("accept a basic situation in an iframe", () => {
+    cy.get("@iframe")
+      .its("0.contentDocument")
+      .should("not.be.empty")
+      .its("body")
+      .as("iframeBody")
+      .then(cy.wrap)
+
+    cy.get("@iframeBody").should("not.be.empty").then(cy.wrap)
```

Comment:
```suggestion
      .as("iframeBody")

    cy.get("@iframeBody").should("not.be.empty")
```
Les wrap ne sont pas nécessaires ici